cfg/validate.go

```diff
@@ -43,29 +44,37 @@ func isValidURL(u string) error {
 	return err
 }
 
+func isValidParallelDownloadConfig(config *Config) error {
+	if config.FileCache.EnableParallelDownloads {
+		if config.FileCache.MaxParallelDownloads == 0 {
+			return errors.New("the value of max-parallel-downloads for file-cache must not be 0 when enable-parallel-downloads is true")
+		}
+		if config.FileCache.WriteBufferSize < CacheUtilMinimumAlignSizeForWriting {
+			return errors.New("the value of write-buffer-size for file-cache can't be less than 4096")
+		}
+		if (config.FileCache.WriteBufferSize % CacheUtilMinimumAlignSizeForWriting) != 0 {
+			return errors.New("the value of write-buffer-size for file-cache should be in multiple of 4096")
+		}
+		if string(config.CacheDir) == "" {
```

Comment:
This suggests - there are two conditions to enable file-cache:
https://github.com/GoogleCloudPlatform/gcsfuse/blob/94a9357fce70096f37a334511bb6d3d58f50148c/cfg/config_util.go#L27

And here we are only checking for CacheDir?

Reply:
Thanks. I based this on `Note that you enable file caching by passing a directory to cache-dir field.` which is mention [here](https://cloud.google.com/storage/docs/cloud-storage-fuse/file-caching#configure).

Is this not accurate? Else I would change this.